### ISSUE TITLE
Zero Gradient Norm and Loss When finetune_language_layers=False in LoRA Fine-Tuning of Qwen2.5 (3B)

### ISSUE THREAD

**Issue by Asma-Bk** [GAD-cell/vlm-grpo#14]
Hello,

I'm currently using your branches to fine-tune Qwen2.5 (3B) for a visual information extraction task. After several experiments, I observed a consistent issue:
When I set finetune_language_layers=False to apply LoRA only on the vision encoder, the training consistently results in:
Zero gradient norm, and Zero loss throughout the entire training process. This seems to suggest that no learning signal is propagating through the trainable layers when the language layers are frozen.

Could you please confirm if this is a known issue ?

**Comment by GAD-cell** [comment]
Hey,
Thank you for the report, thats strange.
I'll check that :)

**Comment by Asma-Bk** [comment]
I tried different values 500 and 2000.
